samples/nrf9160/modem_shell/boards/nrf9160dk_nrf9160_ns.overlay

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Could we put this to a separate commit where we refer to the commit bcf3e8c1f2b70fb791cee7325e1a1d331ee6db94 that removed it accidentally?